documentation/testing_plans.md

```diff
@@ -31,17 +31,11 @@ Before you can write and run tests, you need to set up the test environment.
 1. Ensure your module has `Gemfile`, `Rakefile`, and `spec/spec_helper.rb`
    files.
 
-1. Install the `bundler` gem with Ruby:
-
-    ```shell
-    gem install bundler
-    ```
-
 1. Install the module's gem dependencies, which include Bolt and other testing
    tools:
 
     ```shell
-    bundle install
+    pdk bundle install
```

Comment:
I haven't tested with bolt plans, but if you go the route of using the PDK then this should be the workflow:

```
> pdk new module foo
> cd foo
> # add tests
> pdk validate  # does a gem install using the PDK env, runs spec_prep, etc
> pdk test unit  # runs all tests, ensuring gem install, etc. equivalent of all the manual rake spec commands
```

Reply:
@jpogran you are absolutely right. I'll update the pull to use full pdk instructions. I wasn't looking at big picture. Thanks :-)